dockerfiles/Dockerfile

```diff
@@ -51,7 +52,8 @@ RUN apt-get update && \
     && cd tmp/ && mkdir fluent-bit \
     && tar zxfv fluent-bit.tar.gz -C ./fluent-bit --strip-components=1 \
     && cd fluent-bit/build/ \
-    && rm -rf /tmp/fluent-bit/build/*
+    && rm -rf /tmp/fluent-bit/build/* \
+    && if [ "$FLB_VERSION" = "1.8.11" ] ; then rm -rf /tmp/fluent-bit/plugins/filter_throttle/throttle.c;  rm -rf /tmp/fluent-bit/plugins/filter_throttle/throttle.h; cp /tmp/filter_throttle_patch/* /tmp/fluent-bit/plugins/filter_throttle/; fi
```

Comment:
The throttle plugin is only useful in this version?

Reply:
Different version with different configuration value processing logic （for example, 1.8.12 is different  with 1.9.3）